decidim-admin/app/views/decidim/admin/participatory_processes/show.html.erb

```diff
@@ -30,11 +30,22 @@
     :slug,
     :short_description,
     :description
-    %>
+  %>
+
   <dt><%= display_label(@participatory_process, :hero_image) %></dt>
-  <dd><%= image_tag @participatory_process.hero_image.url.to_s %></dd>
+  <dd>
+    <% if @participatory_process.hero_image %>
+      <%= image_tag @participatory_process.hero_image.url.to_s %>
```

Comment:
There's no default image on the uploader so unless we set up one, it will break.